src/nrnpython/nrnpy_nrn.cpp

```diff
@@ -738,15 +739,13 @@ static PyObject* NPySecObj_pt3dclear(NPySecObj* self, PyObject* args) {
     Section* sec = self->sec_;
     CHECK_SEC_INVALID(sec);
     int req = 0;
-    Py_ssize_t narg = PyTuple_GET_SIZE(args);
-    if (narg) {
-        if (!PyArg_ParseTuple(args, "i", &req)) {
-            return NULL;
-        }
+    nb::tuple args_(args);
+    if (args_.size()) {
+        req = nb::cast<int>(args_[0]);
```

Comment:
If the argument is not int the result will be different (return NULL vs Exception). Will it work the same from the user side?